.github/workflows/sync_repos.yaml

```diff
@@ -25,7 +25,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Harden Runner
-        uses: step-security/harden-runner@63c24ba6bd7ba022e95695ff85de572c04a18142 # v2.7.0
+        uses: step-security/harden-runner@17d0e2bd7d51742c71671bd19fa12bdc9d40a3d6 # v2.8.1
         with:
           egress-policy: audit
```

Comment:
Can we narrow this?

Reply:
I don't see it at https://app.stepsecurity.io/github/carbon-language/carbon-lang/actions/runs